src/dotnet/UI.Blazor/Pages/WalkThroughTestPage/WalkThroughTestPage.razor

```diff
@@ -0,0 +1,45 @@
+@page "/test/walk-through"
+@using ActualChat.UI.Blazor.Services
+@using System.Text
+
+<MainHeader>Walk-Through Test Page</MainHeader>
```

Comment:
please, check for admin right, like in
https://github.com/Actual-Chat/actual-chat/blob/1f02b04f4a6cce529ee067302640169912981295/src/dotnet/Audio.UI.Blazor/Pages/AudioPlayerTestPage/AudioPlayerTestPage.razor#L8